libraries/AP_GPS/AP_GPS.cpp

```diff
@@ -1034,24 +1034,32 @@ void AP_GPS::Write_DataFlash_Log_Startup_messages()
 
 /*
   return the expected lag (in seconds) in the position and velocity readings from the gps
+  return true if the GPS hardware configuration is known or the delay parameter has been set
  */
-float AP_GPS::get_lag(uint8_t instance) const
+bool AP_GPS::get_lag(uint8_t instance, float &lag_sec) const
 {
     // return lag of blended GPS
     if (instance == GPS_MAX_RECEIVERS) {
```

Comment:
Small nitpick that this should be ```GPS_BLENDED_INSTANCE``` rather then ```GPS_MAX_RECEIVERS```. I know it was a pre existing bug, but it's meant to improve the readability of the code.